### PR TITLE
Clown Curse

### PR DESCRIPTION
**Pull request by Arthurdentist**
I wanted to throw this up now for feedback and comment although it isn't finished yet. 

This is the counterpart to the French Curse. It curses the victim with Clown clothes that cannot be removed or acided. It is essentially a weaker cluwne curse.

Specific issues: 
- I want the spell to confer the clumsiness gene and I haven't quite figured that out yet
- I would like this spell to also cause some sort of honking but again I haven't got a specific system for that yet.

Any other comments and thoughts, balance etc would be greatly appreciated.

### CODE REVIEW

File: code/modules/spells/targeted/clowncurse.dm [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		magicclownunder.unacidable = 1
+		magicclownshoes.unacidable = 1
+		magicclownmask.can_flip = 0   //no pushing the mask up off your face
+		var/obj/old_mask = target.wear_mask
+		var/obj/old_uniform = target.w_uniform
+		var /obj/old_shoes = target.shoes
+		target.equip_to_slot(magicclownmask, slot_wear_mask)
+		target.equip_to_slot(magicclownunder, slot_w_uniform)
+		target.equip_to_slot(magicclownshoes, slot_shoes)
+		del(old_mask)
```
Comment (ComicIronic) [inline code/modules/spells/targeted/clowncurse.dm]: qdel these

File: code/modules/spells/targeted/clowncurse.dm [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		del(old_uniform)
+		del(old_shoes)
+		flick("e_flash", target.flash)
+		mutations = list(M_CLUMSY)
```
Comment (ComicIronic) [inline code/modules/spells/targeted/clowncurse.dm]: I believe you can do
`target.dna.SetSEState(CLUMSYBLOCK, 1)`
`target.dna.domutcheck()`

**Comment by AutismSpider** [comment]
So they honk when they talk then? or do stuff?

**Comment by DrCelt** [comment]
And I thought the mime curse was powerful.

Glorious. add this soon.

**Comment by Exxion** [comment]
That last commit corrected the actual learned spells but not the messages given when you learn them.

**Comment by Arthurdentist** [comment]
Should be sorted now.

File: code/modules/spells/targeted/clowncurse.dm [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		var/obj/old_uniform = target.w_uniform
+		var /obj/old_shoes = target.shoes
+		target.equip_to_slot(magicclownmask, slot_wear_mask)
+		target.equip_to_slot(magicclownunder, slot_w_uniform)
+		target.equip_to_slot(magicclownshoes, slot_shoes)
+		qdel(old_mask)
+		qdel(old_uniform)
+		qdel(old_shoes)
+		flick("e_flash", target.flash)
+		target.mutations.Add(M_CLUMSY)
```
Comment (d3athrow) [inline code/modules/spells/targeted/clowncurse.dm]:
~~~
		target.dna.SetSEState(CLUMSYBLOCK,1)
		genemutcheck(target,CLUMSYBLOCK,null,MUTCHK_FORCED)
		target.update_mutations()
~~~
To give them clumsy

Comment (Arthurdentist) [inline code/modules/spells/targeted/clowncurse.dm]: target.mutations.add(M_CLUMSY) is working fine from my own tests

Comment (ComicIronic) [inline code/modules/spells/targeted/clowncurse.dm]: If you're just going to add the mutation, use the framework already in place for genetic spells.
You set the spell's mutations to `list(M_CLUMSY)`

**Pull request merged**